content/docs/components/metadata.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "Metadata"
+description = "Tracking and managing metadata of machine learning workflows in Kubeflow"
+weight = 5
++++
+
+The goal of the [Metadata](https://github.com/kubeflow/metadata) project is to help Kubeflow users understand and manage their machine learning workflows by tracking and managing the metadata of workflows.
+
+## Installation
+
+The Metadata component is installed by default for Kubeflow versions >= 0.6.
+
+If you want to install the latest version of the Metadata component or install it as an application of your Kubernetes cluster, you can follow these steps.
+
+1. Download Kubeflow manifests repository.
+```
+git clone https://github.com/kubeflow/manifests
+```
+
+2. Run the following commands in the manifest repository to deploy services of Metadata component.
+```
+cd metadata/base
+kustomize build . | kubectl apply -n kubeflow -f -
+```
+
+## Python Library
+
+A [Python library](https://github.com/kubeflow/metadata/tree/master/sdk/python#python-client) is published for logging metadata.
```

Comment:
Can you make this a bit more active with: `The Metadata project provides a [Python library](https://github.com/kubeflow/metadata/tree/master/sdk/python#python-client) for logging metadata.`

Also, nit that's unrelated to this PR: I thought it was a bit inconvenient that the [python library readme](https://github.com/kubeflow/metadata/tree/master/sdk/python#sample-usage) doesn't have the installation instructions on it. I know you can find them if you go to `demo.ipynb` but it would probably be helpful for users who arrive at that GitHub page if you have `pip install kfmd` listed somewhere in README.md

Reply:
Makes sense. I will update the github README with installation guide. Thanks